client/datatypes.go

```diff
@@ -2,9 +2,9 @@
 // User Types
 //
 // Generated with goagen v0.2.dev, command line:
-// $ goagen
+// $ goagen.exe
```

Comment:
Not only, also workitem.go, etc.
As for the 2 commits: I try to always commit _all_ changes I have in my workspace. It is the only way to make sure that a _consistent_ set of changes is shared with others. 
If the changes are independent, do it one after the other. If they are dependent, they should be in a single commit.